.changeset/eighty-roses-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+'@wanews/nx-pulumi': major
+---
+
+Deprecate buildTarget and additionalBuiltTargets, and combines them into buildTargets
```

Comment:
Say `BREAKING:`

Then add instructions to migrate. 

In fact you can automate the migration if you want with NX